project-plan.md

```diff
@@ -109,9 +110,37 @@ already being used. This means that we will use:
 * JUnit for unit tests;
 * TravisCI for continuous integration.
 
-# Quality assurance
+## Quality assurance
+
+There are two types of qualit to be assured:
+
+1. Quality of the code;
+2. Quality of the end product.
+
+### Quality of the code
+
+The quality of the code will be assured by applying proper software development
+practices, such as unit testing, continuous integration, code reviews and static
+analysis. We will strive to uphold an architecture TODO supported by proper
+application of the (correct) design patterns, resulting in low coupling and high
+cohesion. For more information on how our development is organised, please see
+the CONTRIBUTING file in our code repositories.
+
+### Quality of the end product
+
+The quality of the product will be assured by asking regularly for feedback from
+the clients. This will be done by discussing status updates, preferably with
+working demos to illustrate and support the progress.
+
+Further towards the end of the development period, we will ask one or two
+members of the client's research group to develop a (toy) language in Spoofax in
+combination with our REPL. This is to gather feedback from someone who has not
```

Comment:
This is too much to ask someone. I think you should rather ask them to use the REPL for one of the _existing_ (smaller/toy) languages, and maybe perform some changes to the language to see how the REPL helps in performing that change.